### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by erikarvstedt**

### CODE REVIEW

File: helper/run-in-nix-env [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
```
Comment (jonasnick) [inline helper/run-in-nix-env:2]: How about we add a header that documents what this script does?

**Review by jonasnick — COMMENTED** [review]
Needs rebase, but looks good to me otherwise.

File: helper/run-in-nix-env (hunk 1 above)
Comment (erikarvstedt) [reply helper/run-in-nix-env:2]: +1. Done.

Comment (erikarvstedt) [reply helper/run-in-nix-env:2]: `run-in-nix-env` was needlessly convoluted, I've cleaned it up.

**Review by jonasnick — APPROVED** [review]
ACK 67a573dae86ae43d30395864351b81ec03c8bef8

**Pull request merged**